### PR TITLE
Update link to community calendar

### PR DESCRIPTION
**Pull request by 2color**
See https://blog.ipfs.tech/2023-01-ipfs-community-calendar/

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

File: docs/community/README.md [hunk 1]
```diff
@@ -25,11 +25,9 @@ Casual chat fan? Here are [all our channels](chat.md). There's getting-started a
 
 ## Community Calendar
 
-If you're an IPFS implementer, consumer, or user, you are welcome to join the community meetings we host regularly.
+If you're an IPFS implementer, application developer, or user, you are welcome to join the community meetings we host regularly.
 
-Stay up-to-date by subscribing to the [**IPFS Community Calendar**](https://calendar.google.com/calendar/embed?src=ipfs.io_eal36ugu5e75s207gfjcu0ae84%40group.calendar.google.com) and joining community meetings such as triage sessions, the IPFS implementers sync, and the IPLD sync.
```
Comment (ElPaisano) [inline docs/community/README.md:30]: _Suggested copy edit:_

If you're an IPFS implementer, application developer, or user, you are welcome to join the community meetings we regularly host. Meetings include triage sessions, the IPFS implementers sync, and the IPLD sync.

For an up-to-date information, subscribe to the [**IPFS Community Calendar**](https://calendar.google.com/calendar/embed?src=ipfs.io_eal36ugu5e75s207gfjcu0ae84%40group.calendar.google.com).

**Review by ElPaisano — APPROVED** [review]
Looks good, link works. Suggesting a minor adjustment in phrasing

**Comment by BigLep** [comment]
For reference, links to additional discussion/notification: https://discuss.ipfs.tech/t/new-year-new-ipfs-community-calendar-ipfs-blog-news/15794/2

**Pull request merged**